tests/unit/lib/FluxibleContext.js

```diff
@@ -86,11 +108,132 @@ describe('FluxibleContext', function () {
                     expect(actionCalls.length).to.equal(1);
                     expect(actionCalls[0].context).to.equal(actionContext);
                     expect(actionCalls[0].payload).to.equal(payload);
-                    expect(actionCalls[0].callback).to.equal(callback);
                     done();
                 };
                 actionContext.executeAction(action, payload, callback);
             });
+
+            it('should call executeAction callback with errors', function (done) {
+                var err = new Error();
+                var action = function (context, payload, callback) {
+                    callback(err);
+                };
+                actionContext.executeAction(action, {}, function (executeActionError) {
+                    expect(executeActionError).to.equal(err);
+                    done();
+                });
+            });
+
+            it('should not swallow callback errors', function (done) {
+                // Error is expected, but will not be catchable. Crudely using domain.
+                var d = domain.create();
+                d.on('error', function () {
+                    console.log('test');
```

Comment:
remove console.log, and maybe assert the error is the `test` error that was thrown.